tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "@tsconfig/node16/tsconfig.json",
+  "compilerOptions": {
+    "preserveConstEnums": true,
+    "sourceMap": true,
+    "noEmit": true,
+    "noImplicitAny": false,
+    "resolveJsonModule": true,
+    "outDir": "./dist"
```

Comment:
We don't need outdir. This config only exists so that `tsc` command uses the correct settings when it checks for errors. We should probably add a `compile` script in `package.json` that only runs the `tsc` command. We should run this before we run the tests in our workflow.